File: work_android_network/ws_android_network_04_3_SSAFYGo_4/src/main/java/com/ssafy/ssafygo/model/service/ReceiptService.java
package com.ssafy.ssafygo.model.service;

import java.util.List;

import com.ssafy.ssafygo.model.dto.Receipt;

public interface ReceiptService {
	// 모든 영수증
    public List<Receipt> findAllReceipt();
    
	// 마지막 영수증
    public Receipt findLastReceipt();

    // 영수증 추가
    public int registerReceipt(Receipt receipt);
}

File: work_android_network/ws_android_network_04_3_leegideok/app/src/main/java/com/ssafy/ssafygo/service/ReceiptService.kt
package com.ssafy.ssafygo.service

import com.ssafy.ssafygo.dto.ReceiptDTO
import com.ssafy.ssafygo.dto.StoreReviewDTO
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.GET
import retrofit2.http.POST
import retrofit2.http.Path

interface ReceiptService {
    @GET("receipt/all")
    fun getReceiptAll(): Call<List<ReceiptDTO>>

    // 마지막 영수증 정보 반환
    @GET("receipt")
    fun getReceipt(): Call<ReceiptDTO>

    // 주문내역 등록
    @POST("receipt")
    fun setReceipt(@Body body: ReceiptDTO): Call<Boolean>

}